.github/workflows/updatenet-update.yml

```diff
@@ -0,0 +1,88 @@
+---
+name: UpdateNet Create
+
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: Name of deployment (updatenet)
+        required: true
+        type: string
+      destination:
+        description: Node image tag, commit sha short or "testnet" or "mainnet"
```

Comment:
I think we don't need neither mainnet nor testnet here since the image we update to would be really an arbitrary node image, so commit sha would just do